share/doc/homebrew/Maintainer-Guidelines.md

```diff
@@ -6,20 +6,118 @@ definitely not a beginner’s guide.
 
 Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
+----
+
+Quick links:
+  [General Guidelines](#general-guidelines) •
+  [Core and Core PRs](#core-and-core-prs) •
+  [Formulae and Formula PRs](#formulae-and-formula-prs)
+
+## General Guidelines
+
 ## Quick Checklist
```

Comment:
You may want to add a title level here because `General Guidelines` above is already at level 2.